test/unittests/Microsoft.ServiceFabric.Diagnostics.Tests/Telemetry/NullMeterProviderTest.cs

```diff
@@ -0,0 +1,52 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using Xunit;
+
+namespace Microsoft.ServiceFabric.Diagnostics.Telemetry
+{
+    public abstract class NullMeterProviderTest
+    {
+        public class CreateMeter : NullMeterProviderTest
+        {
+            [Fact]
+            public void ReturnsNullMeterForNoDimension()
+            {
+                NullMeterProvider<long> meterProvider = new NullMeterProvider<long>();
+                Assert.True(meterProvider.CreateMeter("namespace", "meterName") is NullMeter<long>);
+            }
+
+            [Fact]
+            public void ReturnsNullMeterForOneDimension()
+            {
+                NullMeterProvider<long> longNullMeterProvider = new NullMeterProvider<long>();
+                Assert.True(longNullMeterProvider.CreateMeter("namespace", "meterName", "stringDimension1") is NullMeter1D<long>);
```

Comment:
I would normally define method parameter values as readonly fields generated by fuzzy, but I guess in this case we'd only save on removing the "s, so it might not be worth the extra lines of code.